src/blinding.cpp

```diff
@@ -456,6 +456,27 @@ ustring blind15_sign(ustring_view ed25519_sk, std::string_view server_pk_in, ust
     return result;
 }
 
+ustring blind_version_sign_request(
+        ustring_view ed25519_sk,
+        uint64_t timestamp,
+        std::string_view method,
+        std::string_view path,
+        std::optional<ustring_view> body) {
+    auto [pk, sk] = blind_version_key_pair(ed25519_sk);
+
+    // Signature should be on `TIMESTAMP || METHOD || PATH || BODY`
+    ustring buf;
+    buf.reserve(10 /* timestamp */ + method.size() + path.size() + (body ? body->size() : 0));
+    buf += to_unsigned_sv(std::to_string(timestamp));
+    buf += to_unsigned_sv(method);
+    buf += to_unsigned_sv(path);
+
+    if (body)
+        buf += *body;
+
+    return ed25519::sign({sk.data(), sk.size()}, buf);
```

Comment:
This call to `sign` seemed quite wrong to me, because the comment of that `sign` function reads:

```C++
/// Takes the Ed25519 secret key (64 bytes, or 32-byte seed).  Returns the blinded public key and
/// private key (NOT a seed).
```
which would make this a bit undesirable.  But, it turns out, the *comment* is wrong, and what that function actually does is generate a new seed by cryptographic hash of the current seed.  And so it's okay.

BUT: we should amend that comment, to read: "Returns the blinded public key and blinded libsodium seed value."